doc/api_break.dox

```diff
@@ -10,11 +10,23 @@ remove them nor change their semantics. Existing code should keep working when t
 to another minor version (e.g. from 2.0 to 2.2), so all extensions of existing classes should be done in a manner that
 third party developers do not need to adjust their code to work properly with newer QGIS releases.
 
-Sometimes, however, we may need to break the API as a result of some code changes. These cases should be only exceptions
-and they should happen only after consideration and agreement of the development team. Backwards incompatible changes
-with too big impact should be deferred to a major version release.
+Sometimes, however, we need to break the API as a result of code changes. These cases are exceptions
+and they happen only after consideration and agreement of the development team.
+Backwards incompatible changes with large impact are postponed to the next major release and tracked in
+https://github.com/qgis/qgis4.0_api/issues
+
+This page maintains a list of incompatible changes that happened in previous releases.
+
+QGIS 3.24        {#qgis_api_break_3_24}
+=========
+
+Additional geometry attributes
```

Comment:
```suggestion
Additional geometry attributes       {#qgis_api_break_3_24_additionalgeometryattributes}
```
I have seen that older changes have such an hashtag... What's the purpose of that? It makes likes an entry in the page index?

Reply:
https://qgis.org/api/api_break.html#qgis_api_break_3_22

Exactly. It produces these anchors